docs/source/en/preprocessing.mdx

```diff
@@ -404,6 +434,31 @@ Here is what the image looks like after the transforms are applied. The image ha
     <img src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/preprocessed_image.png"/>
 </div>
 
+<Tip>
+
+For tasks like object detection, semantic segmentation, instance segmentation, and panoptic segmentation, ImageProcessor
+offers post processing methods. These methods convert model's raw outputs into meaningful predictions such as bounding boxes,
+or segmentation maps.
+
+</Tip>
+
+### Pad
```

Comment:
Add a blank line after the section header to correctly render the links below :)

Reply:
Thanks! I need to get used to the blank lines after headers. Fixed in 2f30d954b314406746b125b5be97cca894a97d17